.github/workflows/qa.yml

```diff
@@ -0,0 +1,22 @@
+name: 🕵️ Quality Assurance
```

Comment:
Can you change this to something like "Deploy to QA"? I know the staging one just says "Staging" but really these workflows should be a bit more verb-based.

Reply:
Updated here and staging